src/vs/workbench/contrib/chat/browser/actions/chatNewActions.ts

```diff
@@ -166,7 +168,28 @@ export function registerNewChatActions() {
 			title: localize2('chat.goBack', "Go Back"),
 			icon: Codicon.arrowLeft,
 		},
-		group: 'navigation'
+		group: 'navigation',
+		order: 1
+	});
+
+	registerAction2(class extends ViewAction<ChatViewPane> {
```

Comment:
I would suggest to move this into `agentSessionActions.ts` and let it instantiate the picker and open the session (see below).